scripts/fund-address.ts

```diff
@@ -157,16 +141,18 @@ async function main() {
   }
 
   // instantiate chain object based on given environment and private key
-  const network = PROTOCOL_CONFIG.networks[environment.network_id]
   const chainOptions = {
-    chainId: network.chain_id,
-    environment: argv.environment,
-    gasPrice: parseGasPrice(network.gas_price),
-    network: environment.network_id,
-    provider: expandVars(network.default_provider, process.env)
+    chainId: environment.network.chain_id,
+    environment: environment.id,
+    gasPrice: environment.network.gasPrice,
+    network: environment.network.id,
+    provider: expandVars(environment.network.default_provider, process.env)
```

Comment:
```suggestion
    provider: expandVars(environment?.network?.default_provider, process.env)
```

Reply:
I actually want this to fail if the environment does not exist or if the network does not exist